README.md

```
@@ -31,90 +87,103 @@ curl -X POST http://your-pi-ip:5050/api/plugins/install \
 
 ---
 
+## Community Contributors
+
+LEDMatrix is open to community plugin contributions! The following plugins were built or contributed by community members:
+
+| Plugin | Contributor | Contribution |
+|--------|-------------|--------------|
+| [UFC Scoreboard](./plugins/ufc-scoreboard/) | [@LegoGuy1000](https://github.com/legoguy1000) | Original UFC/MMA implementation ([PR #137](https://github.com/ChuckBuilds/LEDMatrix/pull/137)) |
+
+Want to see your plugin here? Check out [3rd Party Plugin Development](#-3rd-party-plugin-development) below or submit a plugin via [Discord](https://discord.gg/uW36dVAtcT).
```

markdownlint-cli2 (0.20.0)

[warning] Link fragments should be valid

(MD051, link-fragments)